lib/embed.dart

```diff
@@ -156,6 +157,19 @@ class Embed {
   Embed(this.options) {
     _initHostListener();
     dialog = Dialog();
+
+    if(localStorage == null) {
+      dialog.showOk(
+          'Missing browser features',
+          'Check that you\'re using a <a href="'
```

Comment:
I would recommend rephrasing this a bit to head-off any misinterpretations by the privacy-conscious. Consider something like:

* "Hey, it looks like we can't access localStorage!"
* "That could be a problem, because we use localStorage to cache the code you put in the editor."
* "Go to this link for details."